pdr_backend/lake/duckdb_data_store.py

```diff
@@ -349,3 +352,105 @@ def get_query_move_table_data(self, from_table, to_table):
         return (
             f"INSERT INTO {to_table.table_name} SELECT * FROM {from_table.table_name};"
         )
+
+    @enforce_types
+    def export_tables_to_parquet_files(
+        self,
+        seconds_between_exports: int,
+        number_of_files_after_which_re_export_db: int,
+    ):
+        export_folder_path = get_export_folder_path(self.base_path)
+        tables = self.query_data("SHOW TABLES")["name"]
+
+        # Ensure export folder exists
+        os.makedirs(export_folder_path, exist_ok=True)
+
+        for table in tables:
+            table_folder_path = os.path.join(export_folder_path, table)
```

Comment:
I think that can be a function of the table? Like we define its csv representation folder only on-demand, we can also define its export path on demand